src/main/java/org/elastos/hive/AuthToken.java

```diff
@@ -25,6 +25,6 @@ public String getAccessToken() {
 
 	public boolean isExpired() {
 		// TODO;
-		return false;
+		return true;
```

Comment:
Why: false -> true ? Need to implement this method.

Reply:
The new commit implements this method.